packages/styling/src/interfaces/ITheme.ts

```diff
@@ -1,15 +1,18 @@
 import { IPalette } from './IPalette';
 import { IFontStyles } from './IFontStyles';
 import { ISemanticColors } from './ISemanticColors';
+import { IThemeSettings } from './IThemeSettings';
 
 export interface ITheme {
   palette: IPalette;
   fonts: IFontStyles;
   semanticColors: ISemanticColors;
+  settings: IThemeSettings;
 }
 
 export interface IPartialTheme {
   palette?: Partial<IPalette>;
   fonts?: Partial<IFontStyles>;
   semanticColors?: Partial<ISemanticColors>;
+  settings?: Partial<IThemeSettings>;
```

Comment:
Should we just have the `isInverted` flag be a top level thing? You could argue that fonts and palette are settings as well.